bot/persistence/resources/init_db.sql

```diff
@@ -24,6 +24,13 @@ CREATE TABLE IF NOT EXISTS ReactionRole(
     PRIMARY KEY (MessageID, Emoji)
 );
 
+CREATE TABLE IF NOT EXISTS Translation(
+    MessageID TEXT NOT NULL,
+    FlagEmoji TEXT NOT NULL,
+    Translation TEXT NOT NULL,
+    PRIMARY KEY (MessageID)
```

Comment:
Non unique primary key. If a message has two translations (eg. french and english), the insertion will break, bc the message ID will be there twice.

Reply:
I added the emojis later on but forgot to change the primary key here. Thank you!